app/core/decorators/performance.py

```
@@ -0,0 +1,14 @@
+import time
+from functools import wraps
+
+
+def measure_performance(func):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)